Microsoft.Toolkit.Uwp.UI.Controls/TokenizingTextBox/TokenizingTextBox.cs

```diff
@@ -70,21 +73,26 @@ protected override void OnApplyTemplate()
             ContextFlyout = menuFlyout;
         }
 
-        private void AutoSuggestBox_CharacterReceived(UIElement sender, CharacterReceivedRoutedEventArgs args)
-        {
-            if (args.Character.ToString() == TokenDelimiter && sender is AutoSuggestBox autoSuggestBox)
-            {
-                AddToken(autoSuggestBox.Text);
-                autoSuggestBox.Text = string.Empty;
-                autoSuggestBox.Focus(FocusState.Programmatic);
-            }
-        }
-
-        private void AutoSuggestBox_KeyDown(object sender, KeyRoutedEventArgs e)
+        private async void AutoSuggestBox_KeyDown(object sender, KeyRoutedEventArgs e)
         {
             switch (e.Key)
             {
                 case VirtualKey.Back:
+                    {
+                        TextBox autoSuggestTextBox = _autoSuggestBox.FindDescendant<TextBox>() as TextBox;
```

Comment:
As discussed, we should cache this.  You can add a Loaded event to the AutoSuggestBox child and then should be able to grab it there.  The textbox shouldn't change out from underneath us vs. the user pressing backspace a bunch.